src/analysis/normed_space/affine_isometry.lean

```diff
@@ -586,3 +588,42 @@ begin
   rw this,
   simp only [homeomorph.comp_is_open_map_iff, homeomorph.comp_is_open_map_iff'],
 end
+
+local attribute [instance, nolint fails_quickly] affine_subspace.to_normed_add_torsor
+local attribute [instance, nolint fails_quickly] affine_subspace.nonempty_map
+local attribute [instance, nolint fails_quickly] affine_subspace.to_add_torsor
+
```

Comment:
Why do you need these?

Reply:
`nonempty_map` is needed for the statement of `restrict_to_equiv`, but it seems that the other two have indeed become irrelevant.